site/sfguides/src/intro_to_machine_learning_with_snowpark_ml_for_python/intro_to_machine_learning_with_snowpark_ml_for_python.md

```diff
@@ -110,7 +110,7 @@ Now, let's create our first Notebook by importing a `.ipynb` file. First, make s
 
 ![Notebook Dropdown](./assets/nb_dropdown.png)
 
-Next, click the gray upload/import .ipynb button, and select [1_sf_nb_snowflake_ml_data_ingest.ipynb](https://github.com/Snowflake-Labs/sfguide-intro-to-machine-learning-with-snowflake-ml-for-python/blob/main/1_sf_nb_snowflake_ml_data_ingest.ipynb) from your local filesystem:
+Next, click the gray upload/import .ipynb button, and select [0_start_here.ipynb](https://github.com/Snowflake-Labs/sfguide-intro-to-machine-learning-with-snowflake-ml-for-python/blob/main/notebooks/0_start_here.ipynb) from your local filesystem:
```

Comment:
It should be: https://github.com/Snowflake-Labs/sfguide-intro-to-machine-learning-with-snowflake-ml-for-python/blob/main/notebooks/0_start_here.ipynb